.vscode/tasks.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "2.0.0",
+    "tasks": [{
+        "label": "CLI build",
+        "type": "shell",
+        "command": "cargo build",
+        "options": {
+            "cwd": "${workspaceRoot}/apps/framework-cli"
+        }
+    }]
+}
```

Comment:
This can be triggered from VS Code's command palette, or referenced from a `.vscode/launch.json`.